src/color-system.mdx

```diff
@@ -0,0 +1,98 @@
+
+
+### Color swatch names
+<limel-example-extended-color-palette />
+
+The color system consists of 20 base colors or "hues". Each hue has five variations: one default, two tints (light and lighter), and two shades (dark and darker); and this is simply how we have named their CSS variables. The `red` color for instance has the following variable names:
+
+`--color-red-lighter` >  `--color-red-light` >  `--color-red-default` >  `--color-red-dark` >  `--color-red-darker`
+
+
+### Contrast swatch names
+<limel-example-contrast-color-palette />
+
+The contrast swatches are used to clarify the generic and neutral UI elements such as backgrounds, containers, sections, and their textual content. Using them wisely will help create a sense of hierarchy as well as improved readability. CSS variable names for contrast swatches start with a prefix of `--contrast-` followed by a number. These numbers start at `100` and incrementally increase by 100 and stop at `1700`, such as:
+
+`--contrast-100` > `--contrast-200` > ··· > `--contrast-1600` > `--contrast-1700`
+
+**Note** that you should never use these swatches to color something, because these swatches will dramatically change their colors and get inverted in dark and light modes.
+
+**Note** that `--color-white` and `--color-black`, which are displayed beside the contrast swatches do not change in dark and light modes. These variables are perfect when used for example in shadows (which remain black even in dark mode) or text colors on colored buttons.
+
+### Dark mode, light mode, usability
+**Note:** don't confuse titles like "dark mode" and "light mode" or "dark/light themes" with these swatches' naming convention. As mentioned above, each swatch -despite its name- already comes with two tones inbuilt into the color system. For example the `--color-red-dark` variable will automatically be displayed as `#d32f2f;` in light mode, and as `#c62828` in dark mode.
+
+<limel-example-dark-light-mode />
+
+The reduced brightness in the dark mode is set with usability in mind, and will help reduce users' eye strain in low-light environments. In a proper design for dark mode, it is not enough to invert the background and text colors. Colors which are bright in light mode, will appear even brighter in dark mode and therefore pop out more from the UI, which can create visual nuisance.
+
+
+## Usage
+### Using CSS variables
+We have not used HEX values to create the CSS variables for each color. They are instead written in RGB digits, without parentheses. For example `--color-blue-default: 33, 150, 243;` (which would be `#2196f3` in HEX). The reason for such a choice is empowering you to write the colors with alpha channel if needed.
+
+Thus to get a solid blue color, you would write this: `color: rgb(var(--color-blue-default));`,
+or like this to get a blue color with 10% opacity: `color: rgba(var(--color-blue-default), 0.1);`
```

Comment:
```suggestion
Thus, to get a solid blue color, you would write `color: rgb(var(--color-blue-default));`.  
To get a blue color with 10% opacity, write `color: rgba(var(--color-blue-default), 0.1);` instead.
```

(Note that there are two spaces at the end of the first line, to preserve the line-break.)